doc/ems-application-guide/src/ems-calling-points/begin-timestep-before-predictor.tex

```diff
@@ -1,3 +1,7 @@
+\section{Begin Zone Timestep Before Init Heat Balance}\label{begin-timestep-before-predictor}
+
+The calling point called ``BeginZoneTimestepBeforeInitHeatBalance'' occurs at the beginning of each timestep before ``InitHeatBalance'' executes but after the weather manager and exterior energy use manager. ``InitHeatBalance'' refers to the step in EnergyPlus modeling when the solar shading, daylighting, and surface heat balances are calculated. This calling point is useful for controlling components that affect the building envelope including surface constructions, window shades, and shading surfaces. Programs called from this point might actuate the building envelope or internal gains based on current weather or on the results from the previous timestep. Demand management routines might use this calling point to operate window shades, change active window constructions, activate exterior shades, etc.
+
 \section{Begin Timestep Before Predictor}\label{begin-timestep-before-predictor}
 
-The calling point called ``BeginTimestepBeforePredictor'' occurs near the beginning of each timestep but before the predictor executes. ``Predictor'' refers to the step in EnergyPlus modeling when the zone loads are calculated. This calling point is useful for controlling components that affect the thermal loads the HVAC systems will then attempt to meet. Programs called from this point might actuate the building envelope or internal gains based on current weather or on the results from the previous timestep. Demand management routines might use this calling point to reduce lighting or process loads, change thermostat settings, operate window shades, etc.
+The calling point called ``BeginTimestepBeforePredictor'' occurs near the beginning of each timestep but before the predictor executes. ``Predictor'' refers to the step in EnergyPlus modeling when the zone loads are calculated. This calling point is useful for controlling components that affect the thermal loads the HVAC systems will then attempt to meet. Programs called from this point might actuate internal gains based on current weather or on the results from the previous timestep. Demand management routines might use this calling point to reduce lighting or process loads, change thermostat settings, etc.
```

Comment:
Does the IO ref need any accompanying changes?

Reply:
Oh - yes, it does. Will add.

Reply:
Oh, hmm.  OK, well if you make this change on the docs then I guess I'll defer to you whether you want to revert the changes to the test files below.  I don't care either way.  At some point we'll want to change the test file(s) to have this new calling point, but because of my concern with getting the IDD frozen, I'm indifferent on whether it is with this PR or not.